include/odp/api/spec/schedule_types.h

```diff
@@ -401,6 +430,31 @@ typedef struct odp_cache_stash_prio_config_t {
  * Schedule group parameters
  */
 typedef struct odp_schedule_group_param_t {
+	/** Group specific priority configuration */
+	struct {
+		/** Minimum priority level for this group
+		 *
+		 *  The entire group specific priority range (from 'min' to
+		 *  'min' + 'num' - 1) must fit within the global range (from
+		 *  odp_schedule_min_prio() to odp_schedule_max_prio()).
+		 *
+		 *  When creating a queue belonging to a schedule group with
+		 *  a group specific priority range, the used scheduler priority
+		 *  level (odp_schedule_param_t::prio) must be within the group
+		 *  specific range.
+		 *
+		 *  The field is ignored if 'num' is 0.
+		 */
+		odp_schedule_prio_t min;
+
+		/** Number of priorities to be supported by this group
+		 *
+		 *  Use 0 to not limit priorities at group level. 0 by default.
```

Comment:
Another way to say this: "When 0, the global priority range (see odp_schedule_config_t::num_prios) is used with this group."

The main point being that, I'd say more explicitly that "the default config is used when 0" - instead of using negation ("to not limit").